src/prefect/cli/deployment.py

```diff
@@ -821,6 +821,16 @@ async def run(
         "--id",
         help=("A deployment id to search for if no name is given"),
     ),
+    job_variables: List[str] = typer.Option(
+        None,
+        "-jv",
+        "--job-variable",
```

Comment:
The corresponding option on `prefect deploy` is `-v`/`--variable`. We'll want to keep it consistent, but I like how explicit `-jv'/`--job-variable` is. Maybe we could create a backlog ticket to update `prefect deploy`?

Reply:
I can definitely do that -- do you think it's worth making the update in place in this PR or is there a CLI option deprecation process we'd want to follow for a change like that?